### PR TITLE
[tracer][dotnet] add `Kinesis` integration to docs

### PR DESCRIPTION
**Pull request by duncanista**
### What does this PR do? What is the motivation?

Adds the mention that `AWSSDK.Kinesis` package is now available for automatic instrumentation.

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

**Review by pierotibou — APPROVED** [review]

**Review by zacharycmontoya — APPROVED** [review]

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ The [latest version of the .NET Tracer][4] can automatically instrument the foll
 | Aerospike                       | `Aerospike.Client` 4.0.0+                                                                            | `Aerospike`          |
 | ASP.NET Core                    | `Microsoft.AspNetCore`</br>`Microsoft.AspNetCore.App`</br>2.0+ and 3.0+                              | `AspNetCore`         |
 | Azure Functions                 | `Microsoft.Azure.Webjobs` 3.0+                                                                       | `AzureFunctions`     |
+| AWS Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```
Comment (buraizu) [inline content/en/tracing/trace_collection/compatibility/dotnet-core.md:67]:
```suggestion
| Amazon Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```

File: content/en/tracing/trace_collection/compatibility/dotnet-framework.md [hunk 2]
```diff
@@ -63,6 +63,7 @@ The [latest version of the .NET Tracer][5] can automatically instrument the foll
 | ASP.NET (including Web Forms)   | built-in                                                                                  | `AspNet`             |
 | ASP.NET MVC                     | `Microsoft.AspNet.Mvc` 4.0+                                                               | `AspNetMvc`          |
 | ASP.NET Web API 2               | `Microsoft.AspNet.WebApi` 5.1+                                                            | `AspNetWebApi2`      |
+| AWS Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```
Comment (buraizu) [inline content/en/tracing/trace_collection/compatibility/dotnet-framework.md:66]:
```suggestion
| Amazon Kinesis                     | `AWSSDK.Kinesis`  3.0+                                                                               | `AwsKinesis`         |
```

**Review by buraizu — CHANGES_REQUESTED** [review]
Just requesting a minor change to the service name per the [AWS docs](https://docs.aws.amazon.com/)

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md (hunk 1 above)
Comment (duncanista) [reply content/en/tracing/trace_collection/compatibility/dotnet-core.md:67]: Should I change every other service then? I was just keeping consistency with the current docs.

File: content/en/tracing/trace_collection/compatibility/dotnet-framework.md (hunk 2 above)
Comment (duncanista) [reply content/en/tracing/trace_collection/compatibility/dotnet-framework.md:66]: Same as previous comment.

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md (hunk 1 above)
Comment (buraizu) [reply content/en/tracing/trace_collection/compatibility/dotnet-core.md:67]: @duncanista thanks for following up, and no worries about other services. There's a separate docs card (DOCS-6255) open to review AWS service names throughout the documentation which should take care of those.

**Review by buraizu — COMMENTED** [review]
@duncanista thanks for following up, I posted a response to your question about the other AWS service names. Could you let me know if/when this is ready to merge into the documentation repo?

File: content/en/tracing/trace_collection/compatibility/dotnet-core.md (hunk 1 above)
Comment (duncanista) [reply content/en/tracing/trace_collection/compatibility/dotnet-core.md:67]: Thanks for the heads up! Committing suggestion!

**Comment by duncanista** [comment]
Ready to merge @buraizu !

**Review by buraizu — APPROVED** [review]

**Comment by buraizu** [comment]
Thanks for applying those changes @duncanista this looks good to me! However, since the required `build_preview` check is failing on the latest update, I'll just have to merge the `master` branch in first (I believe this is related to the Korean language documentation going live yesterday).
I'll merge into documentation as soon as possible after merging in `master` (usually the checks take about 15 - 20 minutes)

**Comment by buraizu** [comment]
@duncanista apologies, it looks like I can't merge in the `master` branch from the Github UI. Could you merge it in locally and push up that change?

**Pull request merged**